tests/test_global.py

```diff
@@ -0,0 +1,80 @@
+import glob
+import pickle
+import os
+import contextlib
+import io
+from satellite_images_nso_tif_model_iterator.tif_model_iterator import (
+    tif_model_iterator,
+)
+from satellite_images_nso_tif_model_iterator.filenames.file_name_generator import (
+    OutputFileNameGenerator,
+)
+import pandas as pd
+
+# All these variables are local!
+model_path = model_path = (
+    "C:/repos/satellite-images-nso-datascience/saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
```

Comment:
Is there a way to make this a relative path, so it can work out of the box? Instead of having to find and place the correct .sav file in some obscure folder path or having to make code changes in order to make it work?

Reply:
these are files which are made in different repos, will make a setting files for it

Reply:
update: pytest seems to work bad with a setting file, for now a todo for better pathing

Reply:
Looks like this works now, nice!